srcDoc/command_line_parameters.rst

```diff
@@ -345,61 +345,61 @@ Test Parameters for Publishing and Subscribing Applications
-   optional ID of the subscriber could beprovied. This argument may be repeated
+   Adds a peer to the peer host address list. If ``-rawTransport`` is used, 
+   you can provide an optional subscriber ID. This argument may be repeated
    to indicate multiple peers.
 
    **Default:**
    ``Not set. RTI Perftest will use the default initial peers (localhost, shared-memory and multicast).``
 
 -  ``-threadPriorities X:Y:Z``
 
-    This Command Line parameter is supported only for the C++ and C++03 API
-    Implementations.
+    This command-line parameter is supported only for the C++ and C++03 API
+    implementations.
 
-    Set the priorities for the application Threads:
-        X -- For the Main Thread, which will be the one sending the data. Also
+    Set the priorities for the application threads:
+        X -- For the Main Thread, which will be the one sending the data, or 
              for the Asynchronous thread if that one is used.
-        Y -- For the Receive Threads, If the -useReadThread is used, also for
+        Y -- For the Receive Threads or, if the -useReadThread is used, for
              the thread created to receive and process data.
         Z -- For the rest of the threads created by the middleware: Event and
              Database Threads.
 
     Three default values: h (high), n (normal) and l (low) can be used
     instead of numbers.
 
-    To see what values can be used for the different threads see
-    *RTI Connext DDS Core Libraries Platform Notes Version 5.3.1*
+    To see what values can be used for the different threads, see
+    *RTI Connext DDS Core Libraries Platform Notes Version 5.3.1*.
 
     - Table 6.7 Thread-Priority Definitions for Linux Platforms
     - Table 8.6 Thread-Priority Definitions for OS X Platforms
     - Table 12.7 Thread-Priority Definitions for Windows Platforms
 
    This parameter is not available when compiling against *RTI Connext DDS
-   Micro* nor for *Raw Transport*.
+   Micro* or using the Raw Transport feature.
 
    **Default:**
    ``Not set. The priority will not be modified.``
 
-Transport Specific Options
+Transport-Specific Options
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 By default, *RTI Perftest* will try to use the transport settings provided via the
-`xml` configuration file. However, it is possible to override these values directly
-by using the `Transport` spececific command-line parameters.
+XML configuration file. However, it is possible to override these values directly
+by using the transport-spececific command-line parameters.
 
 -  ``-transport <TRANSPORT NAME>``
 
    Set the transport to be used. The rest of the transports will be disabled.
 
    | **Options Pro:** ``UDPv4``, ``UDPv6``, ``SHMEM``, ``TCP``, ``TLS``, ``DTLS`` and ``WAN``.
-   | **Default Pro:** ``Transport defined in the XML profile. (UDPv4 and SHMEM if no changes).``
+   | **Default Pro:** ``Transport defined in the XML profile (UDPv4 and SHMEM if no changes).``
 
```

Comment:
Javi wrote in https://docs.google.com/document/d/1NlepFutM-DFr94KwqYubaQZGoHUzoma9uEc0ouFIp1k/edit?ts=5d35b7e4#:
- I meant that is UDPv4 + SHMEM if the xml profile is not changed.

I still don't understand how to provide two transport names. Does <TRANSPORT NAME> mean that only one transport name is accepted?

Reply:
You cannot provide 2 transport names via Command line, but by default you will have 2 transport enabled: SHMEM and UDPv4, which are Connext DDS default behavior